webui/common/.eslintignore

```diff
@@ -0,0 +1 @@
+**/*.js
```

Comment:
Can you add a newline to the end of this file?

Reply:
updated